### PR TITLE
[CI][Windows][linux] Fix build with wxWidgets 3.2.0

### PR DESCRIPTION
**Pull request by AenBleidd**
Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

**Review by talregev — CHANGES_REQUESTED** [review]
The CI will fail. This is not  the fix for the broken master. Please fix it.

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get -qq update
-          sudo apt-get install -y libftgl-dev freeglut3-dev libcurl4-openssl-dev libxmu-dev libxi-dev libfcgi-dev libxss-dev libnotify-dev libxcb-util0-dev libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev libwebkit2gtk-4.0-dev p7zip-full libxxf86vm-dev ocl-icd-opencl-dev zip
+          sudo apt-get install -y libftgl-dev freeglut3-dev libcurl4-openssl-dev libxmu-dev libxi-dev libfcgi-dev libxss-dev libnotify-dev libxcb-util0-dev libgtk-3-dev libsecret-1-dev libgcrypt20-dev libsystemd-dev libwebkit2gtk-4.0-dev p7zip-full libxxf86vm-dev ocl-icd-opencl-dev zip libx11-dev
```
Comment (talregev) [inline .github/workflows/linux.yml:35]: This is not correct.

Comment (AenBleidd) [reply .github/workflows/linux.yml:35]: What's exactly not correct? Could you please provide more details?

Comment (talregev) [reply .github/workflows/linux.yml:35]: no. It not correct. and you should remove it.

**Comment by AenBleidd** [comment]
Merging this without approval since this is a trivial fix and it unblocks merges to master.

**Pull request merged**